templates/tag.html

```diff
@@ -0,0 +1,197 @@
+                  <div class="ml-4">
+                    <div class="annotation">word 1</div>
+                    <div class="annotation">word 2</div>
+                  </div>
+                </div>
+                <hr>
+                <div class="list-group">
+                  <h4 class="ml-2">Label 2</h4>
+                  <div class="ml-4">
+                    <div class="annotation">word 1</div>
+                    <div class="annotation">word 2</div>
+                    <div class="annotation">word 3</div>
+                  </div>
+                </div>
+                <hr>
+                <div class="list-group">
+                  <h4 class="ml-2">Label 3</h4>
+                  <div class="ml-4">
+                    <div class="annotation">word 1</div>
+                  </div>
+                </div>
+                <hr>
+                <div class="list-group">
+                  <h4 class="ml-2">Label 4</h4>
+                  <div class="ml-4">
+                    <div class="annotation">word 1</div>
+                    <div class="annotation">word 2</div>
+                    <div class="annotation">word 3</div>
+                    <div class="annotation">word 4</div>
+                    <div class="annotation">word 5</div>
+                  </div>
+                </div>
+                <hr>
+                <div class="list-group">
+                  <h4 class="ml-2">Label 5</h4>
+                  <div class="ml-4">
+                    <div class="annotation">word 1</div>
+                    <div class="annotation">word 2</div>
+                    <div class="annotation">word 3</div>
+                    <div class="annotation">word 4</div>
+                    <div class="annotation">word 5</div>
+                    <div class="annotation">word 6</div>
+                  </div>
+                </div>
+                <div class="list-group">
+                  <h4 class="ml-2">Label 6</h4>
+                  <div class="ml-4">
+                    <div class="annotation">word 1</div>
+                    <div class="annotation">word 2</div>
+                    <div class="annotation">word 3</div>
+                    <div class="annotation">word 4</div>
+                    <div class="annotation">word 5</div>
+                    <div class="annotation">word 6</div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- /.col-lg-2 -->
+    </div>
+    <!-- /.row -->
+  </div>
+  <!-- /.container -->
+
+  <!-- Footer -->
+  <footer class="py-2 bg-dark">
+    <div class="container">
+      <p class="m-0 text-center text-white">Copyright &copy; TAG (Temple University) 2019</p>
+    </div>
+    <!-- /.container -->
+  </footer>
+
+  <!-- Bootstrap core JavaScript -->
+  <script src="http://code.jquery.com/jquery-3.4.1.js" integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU="
```

Comment:
change to "https://code.jquery.com/......".  apparently it won't run on the server this way